.github/workflows/autofix.yml

```
@@ -0,0 +1,24 @@
+name: autofix.ci
+on:
+  workflow_call:
+  pull_request:
+  push:
+    branches: [ "main" ]
+permissions:
+  contents: read
+
+
+jobs:
+  autofix:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Install uv
+        uses: astral-sh/setup-uv@v5
+        with:
+          enable-cache: true
+      - run: uvx ruff check --fix-only .
```

```suggestion
      - run: uvx ruff check --fix .
```

When we use `--fix` instead of `--fix-only`, the autofix action will fail when there's a non-fixable linting error.